### ISSUE TITLE
Send single article (as plain text / HTML) to...

### ISSUE THREAD

**Issue by rfc2822** [kiwix/kiwix-android#2637]
**Is your feature request related to a problem? Please describe.**
Often when I use Kiwix to look up some information, I'd like to share this information with friends, for instance over instant messaging.

**Describe the solution you'd like**
In the article view, a "Share/Send to" action would be really helpful. It could provide the page contents as HTML/text using standard Android sharing, so that other apps like an IM client could use them.

**Describe alternatives you've considered**
At the moment, I use copy/paste or, more often, look up the same information in online Wikipedia again and then send this URL.

**Additional context**
If online Wikipedia is not accessible, sharing a single article would especially be useful.

**Comment by s-ayush2903** [comment]
It'll be a nice to have feature, thanks for filing!

**Comment by jackq97** [comment]
If I'm understanding this correctly we want to add the share button in the bottom bar of article view and then share the contents of the article as text , perhaps we can utilize the already written tts function that extracts text for speak aloud.

can you assign this issue to me.

**Comment by MohitMaliFtechiz** [comment]
@jackq97 Before starting the coding effort we should agree on the feature on how it should be implemented.

There are already 5 buttons in the bottom bar, including one more button it is not a good idea IMO. It would not look good at the bottom bar also the bottom bar UI is very simple to use we should not complicate that.

The better place for this option is the `overflow` menu where other options are placed e.g. "Take Notes", "Random article", etc. @kelson42 What do you think?

**Comment by kelson42** [comment]
@MohitMaliFtechiz We had a "share article" in rhe past, right?

**Comment by MohitMaliFtechiz** [comment]
> @MohitMaliFtechiz We had a "share article" in rhe past, right?

@kelson42 AFAIK there was not an option of sharing the article.
